src/util/luxon-utils.ts

```diff
@@ -9,9 +9,16 @@ export type LuxonValue<IsValid extends boolean> = DateTime<IsValid> | Duration<I
 export type LuxonFormat = Brand<string, "LuxonFormat">;
 
 /**
- * @param value - the {@link LuxonValue} to check.
- * @param message - the message to use in the error.
- * @throws error if value is invalid.
+ * Asserts that the provided Luxon value is valid.
+ *
+ * This function checks whether a Luxon date, duration, or interval is valid. If the value is invalid,
+ * it throws an error with a message combining a custom header (or default constructor name) with the
+ * value's invalid reason and, if available, its invalid explanation.
+ *
+ * @param value - The Luxon object to validate.
+ * @param message - Optional custom header for the error message.
+ *
+ * @throws {Error} If the provided value is invalid.
```

Comment:
```suggestion
 * @param value - The Luxon object to validate.
 * @param message - Optional custom header for the error message.
 * @throws {Error} If the provided value is invalid.
```
The project's JSDoc ESLint rules flags extra lines between tags as errors.